components/protocol/params.go

```diff
@@ -27,8 +27,8 @@ type ParametersDatabase struct {
 	Engine           string `default:"rocksdb" usage:"the used database engine (rocksdb/mapdb)"`
 	Path             string `default:"testnet/database" usage:"the path to the database folder"`
 	MaxOpenDBs       int    `default:"10" usage:"maximum number of open database instances"`
-	PruningThreshold uint64 `default:"360" usage:"how many confirmed slots should be retained"`
-	DBGranularity    int64  `default:"1" usage:"how many slots should be contained in a single DB instance"`
+	PruningThreshold uint64 `default:"16384" usage:"how many confirmed slots should be retained"`
```

Comment:
I would keep at least 30 days tbh, if not even more.
With hornet we have 30GB as a default DB size now if we prune by size.

But for our tests now I guess 2 days is ok.

Reply:
I would like this to be the "minimum" for a reasonable network: hence the choice; I didn't run any number, I've set this purely by components' minimal need.